model/src/main/kotlin/OrtResult.kt

```diff
@@ -94,6 +94,10 @@ data class OrtResult(
 
     private data class ProjectEntry(val project: Project, val isExcluded: Boolean)
 
+    /**
+     * A map of projects and their excluded state. Calculating this map once brings massive performance improvements
+     * when querying projects in large analyzer results.
+     */
     private val projects: Map<Identifier, ProjectEntry> by lazy {
         log.info { "Computing excluded projects which may take a while..." }
```

Comment:
Unrelated, but now that I see this again: Wouldn't it make sense to turn the "Computing excluded projects which may take a while..." stuff into `perf` logging so you can also capture these in Splunk?

Reply:
Is `perf` logging output when running with `--info` or `--debug` ?

Reply:
`perf` is exactly in between, so it's output with `--debug`, but not with `--info`. Or, obviously, you use straight `--perf` (which includes `--info`).

Reply:
I think the way it's done currently is to warn the user that this might take a while, but this is not true anymore. In the very large test project I used it took 0.7 seconds to compute both the package and projects. It could make sense to replace it with a single log statement that prints the duration, but I would tend to just remove these log statements.

Reply:
> I would tend to just remove these log statements.

👍🏻

Reply:
We had discussed this same thing recently.
I was against it and still are, as it has potential to save a lot of time.
Keeping the log statement is not expensive.

Reply:
But should we replace it with a single statement in perf level then which prints the duration?

Reply:
The duration can only be print after it's finished, so it would defeat the purpose it was added.

Reply:
I won't block any change removing or adjusting the comment.

Reply:
> The duration can only be print after it's finished, so it would defeat the purpose it was added.

Well, of course you'd add the duration to the concluding log statement. Here's what I propose: https://github.com/oss-review-toolkit/ort/pull/3443

Reply:
@sschuberth  this was just how I interpreted "replace it with a single statement".